src/reporters/json.ts

```diff
@@ -298,11 +312,17 @@ export default class JSONReporter extends BaseReporter {
       journey: {
         name: journey.name,
         id: journey.id,
+        ...(type === 'journey/end' ? { status: payload.status } : {}),
```

Comment:
you can just leave it as 
```
journey: {
   name: journey.name,
   id: journey.id,	       
   status: payload.status,
}
```

This would work only for `journey/end` and `step/end`

Reply:
we only need status for journey/end and step/end, for others it doesnt make sense to add status.

Reply:
Reverted back my suggestion.